make/tests

```diff
@@ -15,21 +15,21 @@ $(LIBGTEST): $(LIBGTEST)(test/gtest.o)
 
 test/gtest.o: $(GTEST)/src/gtest-all.cc
 	@mkdir -p test
-	$(COMPILE.c) -O$O $(CFLAGS_GTEST) $< $(OUTPUT_OPTION)
+	$(COMPILE.cc) -O$O $(CFLAGS_GTEST) $< $(OUTPUT_OPTION)
 
 ##
 # Rule for building a test
 ##
 test/%_test.o : test/%_test.cpp
 	@mkdir -p $(dir $@)
-	$(COMPILE.c) -O$O $(CFLAGS_GTEST) $< $(OUTPUT_OPTION)
+	$(COMPILE.cc) -O$O $(CFLAGS_GTEST) $< $(OUTPUT_OPTION)
```

Comment:
We can now get rid of this! Awesome!

We should be able to use the built-in macros if we add a target-specific variables. https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html

Did you want to try to tackle it? If you want me to try, I'll make some time later this week.